test/TestExpectations.json

```diff
@@ -2051,6 +2051,12 @@
     "parameters": ["cdp", "chrome"],
     "expectations": ["FAIL"]
   },
+  {
+    "testIdPattern": "[keyboard.spec] Keyboard should report shiftKey",
+    "platforms": ["darwin"],
+    "parameters": ["firefox", "webDriverBiDi"],
+    "expectations": ["FAIL"]
```

Comment:
This test seems to be incorrect and may reveal an issue with CDP? I've filed https://github.com/puppeteer/puppeteer/issues/11599 for investigation.